uploadPhoto.py

```diff
@@ -0,0 +1,45 @@
+import cv2
+import numpy as np
+import json
+import requests
+from datetime import datetime
+
+
+headers = {
+    "Authorization": "Bearer ##secret key##"}
```

Comment:
nebylo by lepší tohle načítat ze souboru? Udělej to stejně jako s cestou přes argparse s nějakou defaltní hodnotou